server/src/test/java/org/elasticsearch/index/query/NestedQueryBuilderTests.java

```diff
@@ -178,7 +178,34 @@ public void testFromJson() throws IOException {
             }""";
 
         NestedQueryBuilder parsed = (NestedQueryBuilder) parseQuery(json);
-        checkGeneratedJson(json, parsed);
+        checkGeneratedJson("""
```

Comment:
Same thing here, add a comment noting that the missing fields are missing because they're defaults.